### ISSUE TITLE
[Feature Request] Add Following tokens to MATT Teleport integration

### ISSUE THREAD

**Issue by ottyn** [Saibot393/Rideable#67]
Feature request to add the ability to check for "following" tokens when using a MATT Teleport tile.  In the same method that "mounted" tokens will auto-teleport with MATT Teleports currently.

**Comment by Saibot393** [comment]
Planning on doing so, will take a bit though, i am currently low on time. I would guess that i can do it within the next two weeks.

**Comment by maplealmond** [comment]
+1 -- Would like to see this for Stairways teleport as well.  If following, follow through the stairs.

**Comment by Saibot393** [comment]
Yeah, the 2 weeks timeframe was a bit to optimistic, sorry. I will still do it, but i don't really have an ETA right now.

**Comment by ottyn** [comment]
Saw the Stairways teleport integration in the 3.1.0 update.  Haven't tested with MATT or Region teleports yet, but it might be a good idea to include them all when you get a chance to work on it @Saibot393   No hurry though.

**Comment by Saibot393** [comment]
Yes, i tried looking into adding compatibility with MATT teleports for this update. The underlying logic for MATT teleports however is quite different from the stairways teleport and as such the compatibility requires a more complex logic which i didn't have time to implement yet (still not sure if there even is an elegant solution here).
